frontend/src/components/7.Result/DataVisual.js

```diff
@@ -10,9 +10,18 @@ import {Pie} from 'react-chartjs-2';
  * 
  */
 function DataVisual(props) {
+  // Modify the options prop to change the chart legend's font colour
+  const chartOptions = {
+    legend: {
+      display: true,
+      labels: {
+        fontColor: 'rgb(255, 255, 255)',
+      },
+    },
+  };
   return (
     <div className='PieChart'>
-      <Pie data={props.data}/>
+      <Pie data={props.data} options={chartOptions} />
```

Comment:
Hi Justin!

Could you please add the following additional prop to the Pie component: `height={250}`.
This will make the Pie chart size a little better, especially once the number of datavalues increases beyond 6 from the work in my latest PR.

Thanks! :)

Reply:
![image](https://user-images.githubusercontent.com/55283692/115316669-e5297880-a1cd-11eb-9541-f4b3fdc32501.png)

Hows that?

Reply:
Perfect